distribution/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2.4'
+services:
+  composer:
+    image: local/osbuild-composer
+    build:
+      context: ..
+      dockerfile: ./distribution/Dockerfile-ubi
```

Comment:
Would it be possible to change the name from Dockerfile-ubi to something like Dockerfile-composer without breaking anything? It would make the naming more consistent.

Reply:
I see it mentioned a couple of times in the repo (`schutzbot/containerbuild.sh` and `tools/clouddot_build_deploy.sh`) and I don't know if it's also used anywhere outside.  I'd be in favour of renaming for consistency, but didn't want to make any unrelated changes.

I'd also be in favour of either:
- Naming them plain `Dockerfile` in a separate directory each (i.e., `distribution/composer/Dockerfile` and `distribution/worker/Dockerfile`).
- Naming them `x.Dockerfile` (i.e., `composer.Dockerfile` and `worker.Dockerfile`).

The former is conventional (in my experience) and it can be used to separate each with their scripts or accompanying files, though we don't seem to have any that are specific to each now.
The latter is also conventional when there are multiple Dockerfiles and helps identify the "type" for editors and browsers (and GitHub).

Reply:
I think we can rename this now that we are actually not deploying from this file. Just naming it `Dockerfile` seems appealing to me, do you lose anything compared to the second suggestion, or will all the tools still identify it correctly?

Reply:
For a follow-up though!